geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java

```diff
@@ -687,41 +675,37 @@ public void run2() throws CacheException {
    */
   void checkCqListenerEvents(VM vm, final String cqName, final int numEvents,
       final int secondsToWait) {
-    vm.invoke(() -> {
-      checkCqListenerEvents(cqName, numEvents, secondsToWait);
-    });
+    vm.invoke(() -> checkCqListenerEvents(cqName, numEvents, secondsToWait));
   }
 
-  void checkCqListenerEvents(final String cqName, final int numEvents,
-      final int secondsToWait) {
-    QueryService qs = CacheServerTestUtil.getCache().getQueryService();
+  void checkCqListenerEvents(final String cqName, final int numEvents, final int secondsToWait) {
+    QueryService qs = getCache().getQueryService();
     CqQuery cq = qs.getCq(cqName);
     // Get the listener and wait for the appropriate number of events
-    CacheServerTestUtil.ControlCqListener listener =
-        (CacheServerTestUtil.ControlCqListener) cq.getCqAttributes().getCqListener();
-    listener.waitWhileNotEnoughEvents(secondsToWait * 1000, numEvents);
+    ControlCqListener listener =
+        (ControlCqListener) cq.getCqAttributes().getCqListener();
+    listener.waitWhileNotEnoughEvents(secondsToWait * 1000L, numEvents);
```

Comment:
Using `TimeUnits.SECONDS.toMillis(secondsToWait)` makes it more readable.

Reply:
Done